src/lists/list-item.jsx

```diff
@@ -407,6 +407,7 @@ const ListItem = React.createClass({
     const {
       innerDivStyle,
       style,
+      ...other,
```

Comment:
Good point. I will work on passing the other spread from the render method today.